This content has moved to https://commcare-cloud.readthedocs.io/en/latest/changelog/0045-upgrade-to-python-39.html
